tests/e2e/utils/helpers.js

```diff
@@ -0,0 +1,68 @@
+const axios = require('axios');
```

Comment:
See previous comment about possibly replacing **axios** w/ **got** to avoid adding another dependency.